movement/io/load_poses.py

```diff
@@ -678,3 +679,102 @@ def _ds_from_valid_data(data: ValidPosesDataset) -> xr.Dataset:
             "source_file": None,
         },
     )
+
+
+def from_mmp_file(
+    file_path: Union[Path, str], fps: Optional[float] = None
+) -> xr.Dataset:
+    """Load pose tracking data from a JSON file with multiple individuals.
+
+    This function expects JSON data in the following format:
+    [
+        {
+            "frame_id": int,
+            "instances": [
+                {
+                    "keypoints": list[list[float]],
+                    "keypoint_scores": list[float],
+                    "bbox": list[float],
+                    "bbox_score": float
+                },
+                ...
+            ]
+        },
+        ...
+    ]
```

Comment:
This infor is useful, but better move it to the "Notes" section of the docstring, or better still add a reference to the corresponding MMPose documentation page where the format is describe. See `from_sleap_file()` for the syntax for "Notes" and "References".